src/node_http2.cc

```diff
@@ -89,7 +89,7 @@ ssize_t Http2Session::OnCallbackPadding(size_t frameLen,
   Context::Scope context_scope(context);
 
 #if defined(DEBUG) && DEBUG
-  CHECK(object->Has(context, env()->ongetpadding_string()).FromJust());
+  // CHECK(object->Has(context, env()->ongetpadding_string()).FromJust());
```

Comment:
I think
`CHECK(object->Has(context, env()->ongetpadding_string().FromJust()));`
is the right one. Without which, the whole expression was evaluated to be a value type, while we expect to have a boolean type for the CHECK.